### PR TITLE
Major Profile Mode improvements

### PR DESCRIPTION
**Pull request by Romualdo666**
## Description
When opening a data.win, all the code in the Temp Profile folder is replaced with the main, and when saving, all the main is replaced with the temp. this, to avoid wrong overwriting. Also, exporting profiles will actually overwrite the already exported profiles, and importing too. Fixed the issue where Importing Profiles wouldn't edit the dissasembly code.
### Caveats
If you had a tab with code opened, and you import THAT code with the "Import Profile" button, it will get overwritten, but visually won't be updated until you close or switch the tab.

### Notes
Sorry if I'm making too many pull request, haha.

### CODE REVIEW

**Comment by VladiStep** [comment]
Your PR also has the old "dogcheck" and some compiler changes that you didn't mention.
You should make separate PRs for them.

**Comment by colinator27** [comment]
> Your PR also has the old "dogcheck" and some compiler changes that you didn't mention. You should make separate PRs for them.

Agreed on this -- currently we have no intention of restoring the "dogcheck" mechanism that confused many people. Also, seems like there's some extraneous compiler changes, as well as some code that maybe should be locked behind a profile mode check (e.g., checking for string comments).

**Comment by Miepee** [comment]
@Romualdo666 could you please revert the dogcheck?

**Comment by Romualdo666** [comment]
aight sure

**Comment by Romualdo666** [comment]
@Miepee that should do it

File: UndertaleModTool/Resources/mus_dance_of_dog.wav
Comment (Miepee) [inline UndertaleModTool/Resources/mus_dance_of_dog.wav:1]: this iss till dogcheck related

File: UndertaleModTool/Resources/spr_tobdogl/0.png
Comment (Miepee) [inline UndertaleModTool/Resources/spr_tobdogl/0.png:1]: this is still dogcheck related

File: UndertaleModTool/Resources/spr_tobdogl/1.png
Comment (Miepee) [inline UndertaleModTool/Resources/spr_tobdogl/1.png:1]: this is still dogcheck related

File: UndertaleModTool/UndertaleModTool.csproj [hunk 1]
```diff
@@ -153,4 +153,33 @@
...
+      <None Update="Properties\Settings.settings">
+        <Generator>SettingsSingleFileGenerator</Generator>
+        <LastGenOutput>Settings.Designer.cs</LastGenOutput>
+      </None>
+    </ItemGroup>
+	<ItemGroup>
+		<Resource Include="Resources\spr_tobdogl\0.png" />
+		<Resource Include="Resources\spr_tobdogl\1.png" />
+		<Resource Include="Resources\mus_dance_of_dog.wav" />
+	</ItemGroup>
```
Comment (Miepee) [inline UndertaleModTool/UndertaleModTool.csproj:184]: this is still dogcheck related

File: UndertaleModTool/Windows/SettingsWindow.xaml
Comment (Miepee) [inline UndertaleModTool/Windows/SettingsWindow.xaml:1]: why is this deleted?

File: UndertaleModTool/MainWindow.xaml [hunk 2]
```diff
@@ -713,5 +715,42 @@
...
+                <Image Grid.Row="1" Grid.Column="1">
+                    <Image.Triggers>
+                        <EventTrigger RoutedEvent="Loaded">
+                            <BeginStoryboard>
+                                <Storyboard RepeatBehavior="Forever">
+                                    <ObjectAnimationUsingKeyFrames Storyboard.TargetProperty="Source"
+                                                   Duration="0:0:0.4">
+                                        <DiscreteObjectKeyFrame KeyTime="0:0:0.00">
+                                            <DiscreteObjectKeyFrame.Value>
+                                                <BitmapImage UriSource="Resources/spr_tobdogl/0.png"/>
```
Comment (Miepee) [inline UndertaleModTool/MainWindow.xaml:739]: this is still dogcheck related

**Review by Miepee — CHANGES_REQUESTED** [review]

**Comment by Romualdo666** [comment]
oh crap i deleted the wrong file my bad. also, didnt know the assets were also needed to get deleted. i'll delete them then.

**Comment by Romualdo666** [comment]
@Miepee this was already a mess so i am just making a new pull request when i have it done, sorry.

**Pull request closed without merging**